_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+title: Highlights from the DBT Community's Coalesce Conference 
+date: 2021-01-25
+tags: data dbt elt etl transformation
+author: Paul Symons
+image: img/blog/coalesce2020/banner.jpg
+---
+
+In this article, I want to share some of my highlights from the recent [Coalesce 2020](https://www.getdbt.com/coalesce) conference, where analytics teams from all over the world shared their experiences about improving and accelerating [Analytics Engineering](https://blog.getdbt.com/what-is-an-analytics-engineer/), and how adopting [DBT](https://www.getdbt.com) has often played a pivotal role. Held as a virtual conference in December 2020 by [FishTown Analytics](https://www.fishtownanalytics.com/), it was presented for, and by, the community of DBT users worldwide.
```

Comment:
I think I need a one line intro to DBT here. Prob before this para...

DBT is a new and exciting paradigm etc...

Why should I get excited?

Reply:
Maybe something to do with the audience... "if you work with data..." etc

Reply:
moved the **DBT - recap** paragraph further up